current-development-phase.md

```diff
@@ -51,9 +51,11 @@ We provide sketches of the basic generation, storage, and use of arbitrary secre
     1. Non-normative note: Given the above properties, we do not achieve integrity of the audit logs in the presence of a cheating key server. Future work may address this concern.
 
 ## Cryptographic Protocol and Implementation Dependencies
-1. We instantiate the asymmetric password-based authenticated key exchange protocol with OPAQUE. We are currently using [opaque-ke](https://docs.rs/opaque-ke/2.0.0-pre.2/opaque_ke/index.html), which currently implements [version 09 of the IETF RFC](https://datatracker.ietf.org/doc/draft-irtf-cfrg-opaque/09/). This library is under active development, as is the IETF draft. An earlier release of this repository has been audited by NCC Group in June 2021. 
-    - [TODO](https://github.com/boltlabs-inc/key-mgmt-spec/issues/21): Select and add configuration details relevant to a cryptographic audit.
-1. TLS 1.3. 
+- We instantiate the asymmetric password-based authenticated key exchange protocol with OPAQUE. We are currently using [opaque-ke](https://docs.rs/opaque-ke/2.0.0-pre.2/opaque_ke/index.html), which currently implements [version 09 of the IETF RFC](https://datatracker.ietf.org/doc/draft-irtf-cfrg-opaque/09/). This library is under active development, as is the IETF draft. An earlier release of this repository has been audited by NCC Group in June 2021. 
+    - This implementation relies on [voprf](https://github.com/novifinancial/voprf), which is tracking [the IETF RFC on OPRFs](https://datatracker.ietf.org/doc/draft-irtf-cfrg-voprf/).
+    - As both of the above RFCs are in flux, we expect ongoing updates.
+    - We currently use the following OPAQUE-3DH configuration: OPRF(ristretto255, SHA-512), HKDF-SHA-512, HMAC-SHA-512, SHA-512, ristretto255, with Argon2 as the KSF.
```

Comment:
Is this the standard way to write out the configuration? I'd prefer to either link to something or add explanatory text to describe what field each of these items satisfies (e.g why are SHA-512 and ristretto255 listed twice?).

Reply:
This is directly from the OPAQUE RFC.

Reply:
they aren't listed twice... but to expand on this it would be something like:
OPRF selection:
  - group: Ristretto, 
  - hash: SHA-512
KDF selection: HKDF-SHA-512
MAC selection: HMAC-SHA-512
Hash selection: SHA-512
Group for 3DH: Ristretto

Does that really add value, though?

Reply:
> This is directly from the OPAQUE RFC.

Okay, then it's fine for me.